web/public/templates/profiles/general-info.dust

```diff
@@ -1,3 +1,13 @@
+<div ng-if="!editMode && !publicChampion">
+  <div class="alert alert-info" ng-if="parentProfile && profile.ownProfileFlag">
+    {@i18n key="Only your username and profile picture are publicly viewable. Other fields are for administration purposes only. "/}
+    <a ng-click="publicProfile()">Click to view your public profile</a>
+  </div>
+  <div class="alert alert-info" ng-if="!parentProfile && (profile.ownProfileFlag  || profile.myChild)">
```

Comment:
Probably, even more when the ng-if conditions are actually opposite :)
Will have a look at it this week, thanks for the head up !